cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -200,6 +200,13 @@
     </description>
   </property>
 
+  <property>
+    <name>zookeeper.client.timeout.millis</name>
+    <value>40000</value>
+    <description>
+      ZooKeeper client timeout in milliseconds
```

Comment:
It is not entirely true. It is more like the max time to wait for the first successful connection to one of the server in a ZK quorum.